Altis update but cooler

Tiny-DM opened the pull request:
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [X] Enhancement

### What have you changed and why?
Updated altis with a few more locations and modified some locations to have more interesting structures / better cover.
Modifications made by Mimieux, I only made minor direct metadata edits
See image for reference: yellow locations were adjusted while red locations are new.
<img width="1397" height="1003" alt="image" src="https://github.com/user-attachments/assets/265364ca-6968-4a92-9e35-f86fd19cc764" />
Only modifications since this image have been to change the new seaport in the center to an outpost and to add some more cover to the stadium buildings.

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: Probably needs a testing run and the vehicle places need to be confirmed to be solid.

Tiny-DM committed:
Update mission.sqm

wersal454 commented:
<img width="1000" height="960" alt="image" src="https://github.com/user-attachments/assets/a22bbbb4-b4bf-4905-9171-d81ab55d07d5" />

Tiny-DM committed:
Update mission.sqm

Tiny-DM commented:
Updated to most recent version. Updated changes map: 
<img width="1439" height="1018" alt="AltisUpdatesMap" src="https://github.com/user-attachments/assets/a779768b-b4bc-4a2c-b489-5079766a6a01" />

> orange is like "heavily" edited, yellow is light and red once again is new places

jaj22 commented:
Changes are pretty good. I was a bit concerned that the building spam was a bit excessive but apparently they're cheap in Arma. The new stuff is mostly added to quiet areas anyway.

There are some other things I'd be inclined to change in Altis. Some areas are land QRF deserts due to land attack distance being a bit short and key outposts lacking vehicle places. The radio towers could do with shifting around and thinning out. Doesn't need doing in this PR though.

**Issues:**
outpost_16: Vehicle marker extremely sloped. Probably needs moving to the other side, with the bunker complex shuffled down the road.

outpost_30: MG tower has too much tilt. Can be forced flat.

outpost_45: Vehicles place is a bit close to the crates. Either move it back a couple of metres or flip it towards the road.

Tiny-DM committed:
Update mission.sqm

Tiny-DM committed:
altis update

jaj22 reviewed (approved):
Edits taken on trust.
